heroku/cli
# migrate(W-14179828): pg-v5: Upgrade pg:settings:auto-explain

This PR moves `pg:settings:auto-explain` to the cli and migrates to TypeScript

## Test Instructions
1. `git checkout jw/migrate-pg-settings-auto-explain && yarn && yarn build && cd packages/cli`
2. `./bin/run pg:settings:auto-explain -a YOUR_APP heroku-postgresql`
3. Observe the output is the same as the existing cli
